proto/rill/admin/v1/api.proto

```diff
@@ -744,7 +752,12 @@ message CreateProjectRequest {
   int64 prod_slots = 8;
   string subpath = 12;
   string prod_branch = 9;
+  // github_url is set for projects whose artifacts are stored in github. This is set to a github repo url.
+  // Either github_url or upload_path should be set.
   string github_url = 10;
+  // upload_path is set for projects whose artifacts are not stored in github but are managed by rill. This is set to a path in gcs.
+  // When upload_path is set github_url, prod_branch, subpath will be empty.
+  string upload_path = 14;
```

Comment:
Good finding.

1. Sounds good.
2. This is error prone in my opinion.
3. How do you feel about parsing `upload_id` to get org and checking if the user passing `upoad_id` is an admin in the org ? Otherwise 1 sounds good to me.

Reply:
I also think in `<org>__<proj>__<uuid>`, `proj>` is unnecessary because projects can be renamed.